test/dlang/install/test

```diff
@@ -8,7 +8,7 @@ set -xeu
 # Test basic usage
 for DIST in xenial
 do
-    export DIST DMD=2.067.1-0
+    export DIST DMD=2.077.1-0
```

Comment:
Same

Reply:
Right, so it is here too. Then I think you need to add another `for` in here, so you can test both.
```sh
for DIST in xenial
do
    for DMD in 2.067.1-0 2.077.1-0
    do
        export DIST DMD
        # ...
    done
done
```